File: src/EventsMicroservice/Events.Domain/EventParticipant.cs
namespace Events.Domain;

public class EventParticipant
{
    public int Id { get; set; }

    public int EventId { get; set; }
    public int UserId { get; set; }
    public DateTime CreateDate { get; set; }
    public int ParticipantStatus { get; set; }
    public int PaymentStatus { get; set; }
}

File: src/foodclub-ui-next/types/event-participant.ts
import { PaymentStatus } from "@/types/payment-status";
import { ParticipantStatus } from "@/types/participant-status";

export type EventParticipantDto = {
    id: string;
    eventId: string;
    userId: string;
    createdAt: string;
    participantStatus: ParticipantStatus;
    paymentStatus: PaymentStatus;
    paymentIntentId: string;
}